frontend/src/container/FormAlertRules/ChartPreview/index.tsx

```diff
@@ -23,6 +23,7 @@ export interface ChartPreviewProps {
 	headline?: JSX.Element;
 	threshold?: number | undefined;
 	userQueryKey?: string;
+	isStepIntervalSelected?: boolean;
```

Comment:
can we have a good name please according to the component used + having some context around it 😅

Reply:
Okay.